### DISCUSSION TITLE
Comments from individual commit

### DISCUSSION THREAD

**Discussion by Craige** [community/community#16320] [Mobile] [answered]
During a pull request, Github web allows you to view an individual commit and see/leave PR feedback from there. The mobile app does not let you leave or see comments from this view.

It would be extremely helpful to be able to do so.

**Comment by amrehman** [comment, marked as answer]
Hi @Craige ! Thanks a lot for reaching out to us! This is definitely a feature gap! 

I am going to create a tracking issue for this on our side and we will prioritise this accordingly! Stay tuned to our future releases :) 🚀
